### PR TITLE
Puppeteer install instructions

### PR DESCRIPTION
**Pull request by strass**
Just a note on how to fix an error I encountered due to not having puppeteer installed

### CODE REVIEW

**Comment by Hypnosphi** [comment]
Weird, `puppeteer` is actually a direct dependency of `@storybook/addon-storyshots`:

https://github.com/storybooks/storybook/blob/master/addons/storyshots/package.json#L27

**Comment by strass** [comment]
@Hypnosphi that is very strange... Do you know if anyone else has experienced the error `Jest: 'it' and 'test' must return either a Promise or undefined.` when trying to run based on the image snapshots alpha instructions

**Comment by thomasbertet** [comment]
Not that I am aware of. Which version of Jest are you using ? 
Did you specify any _before_ hook ?

**Comment by strass** [comment]
@thomasbertet I am using the latest version of CRA, no before hooks (you mean on the script invocation, right?)

**Review by ndelangen — APPROVED** [review]

File: addons/storyshots/README.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ Internally here are the steps:
 - Browses each stories (calling _<http://localhost:6006/iframe.html?...>_ URL),
 - Take screenshots & save all images under _\_image_snapshots\__ folder.
 
+**Puppeteer is not installed by default** as that would increase install times for all storyshot users. `yarn add puppeteer --dev` will save it to your package.json. This will fix the error `Jest: 'it' and 'test' must return either a Promise or undefined.`
```
Comment (Hypnosphi) [inline addons/storyshots/README.md]: > Puppeteer is not installed by default

@ndelangen this is just not true

**Pull request closed without merging**